graylog2-server/src/main/java/org/graylog2/streams/StreamRouterEngine.java

```diff
@@ -180,7 +216,7 @@ public String getFingerprint() {
 
             for (final StreamRule streamRule : stream.getStreamRules()) {
                 try {
-                    final Rule rule = new Rule(stream, streamRule);
+                    final Rule rule = new Rule(stream, streamRule, null);
```

Comment:
The reason was because of the reuse in the test matching logic. It is misleading to pass any value there. Null checking is a pita too. I will change to a primitive and add a comment.